models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
```

Comment:
Hi @kegan, looks like you added this `models.py` and `admin.py` to the root. Don't think these should be here, can you remove them?

Reply:
Hi @kegan, would you be able to look into the above?

Reply:
@svleeuwen My apology for the super late reply 😬 I have fixed it by removing both `models.py` and `admin.py` from the commits.

Hope you can check if this is ok.